services/quests/src/actions/Quest.tsx

```diff
@@ -170,8 +170,9 @@ function createDocNotes(model: any) {
   return str;
 }
 
-function loadQuestFromAPI(user: UserState, docid: string): Promise<{data: string, notes: string, metadata: any}|null> {
-  return fetch(API_HOST + '/qdl/' + docid, {
+function loadQuestFromAPI(user: UserState, docid: string, edittime: Date): Promise<{data: string, notes: string, metadata: any, edittime: Date}|null> {
+  console.log(edittime.getTime());
```

Comment:
Rm the extra console logs here and elsewhere

Reply:
Done.